XRTK-Core/Packages/com.xrtk.core/Runtime/Providers/CameraSystem/BaseCameraDataProvider.cs

```diff
@@ -156,55 +156,19 @@ public override void Initialize()
         {
             base.Initialize();
 
-            if (CameraRig == null)
-            {
-                if (CameraCache.Main.transform.parent.IsNull())
-                {
-                    var rigTransform = new GameObject().transform;
-                    CameraCache.Main.transform.SetParent(rigTransform);
-                }
-
-                CameraRig = CameraCache.Main.transform.parent.gameObject.EnsureComponent(cameraRigType) as IMixedRealityCameraRig;
-                Debug.Assert(CameraRig != null);
-            }
-
-#if XRTK_USE_LEGACYVR
-            ApplySettingsForDefaultHeadHeight();
-#else
-            // We attempt to intialize the camera tracking origin, which might
-            // fail at this point if the subystems are not ready, in which case,
-            // we set a flag to keep trying.
-            trackingOriginInitialized = SetupTrackingOrigin();
```

Comment:
This was in Initialize because it only needs to happen once. Now that you moved it to OnEnable it will run everytime the service is enabled potentially. It needs to be updated, refactored so that it doesn't run tracking origin initialization if it is already initiailzed.

Reply:
I'll re-evaluate, however, this was raising issues in the editor when you hit play and spamming the console because the tracking type was "unknown"

Reply:
I've reverted the change and implemented a check for the MockHMD which reports as a system but does not support tracking.  Works as expected now.